fbw-a380x/src/systems/systems-host/systems/FlightWarningSystem/FwsCore.ts

```diff
@@ -2438,24 +2467,23 @@ export class FwsCore {
     this.autoPilotInstinctiveDiscPressedPulse.write(false, deltaTime);
 
     // A/THR OFF
-    const aThrEngaged = this.autoThrustStatus.get() === 2;
+    const aThrEngaged = this.autoThrustStatus.get() === 2 || this.autoThrustMode.get() !== 0;
```

Comment:
What is the purpose of `this.autoThrustMode.get() !== 0`?

Reply:
For some of the A/THR modes such as MAN FLEX, MAN MCT etc. A/THR is armed but not engaged, but we still don't want to trigger the memo or fault. This ensures we only trigger the memo and/or fault if A/THR mode is NONE. Might be partly redundant to Status being 2, but didn't want to change the trigger conditions too much right now